### PR TITLE
fix: incomplete events

### PR DESCRIPTION
**Pull request by danielattilasimon**
Add missing `boldAccrued` to `SnapshotVotes`.
Add missing `vetos` to `SnapshotVotesForInitiative`. Add missing success flags when calling hooks.
Add `indexed` to select events.

Had to slightly refactor `_allocateLQTY()` to avoid running into "stack too deep".

**Fixes CS-V2Gov-040.**

### CODE REVIEW

**Review by bingen — APPROVED** [review]
Looks good!
(Although the refactor into `_allocateLQTYToInitiative` was hard to follow on git diff :grimacing: )

**Comment by danielattilasimon** [comment]
> Looks good! (Although the refactor into `_allocateLQTYToInitiative` was hard to follow on git diff 😬 )

Sorry about that 😅 
I guess just the memory struct would be enough to avoid stack-too-deep, let me try that.

**Comment by danielattilasimon** [comment]
Ugh, it's not that easy, then I have to put a lot more stuff in memory and it gets even more expensive gas-wise.

**Comment by danielattilasimon** [comment]
Closing in favor of #94
